README.md

```diff
@@ -1,61 +1,64 @@
 <p align="center">
-  <a
-    href="https://github.com/SecureSECODAO/diamond-governance/blob/master/LICENSE"
-    alt="License"
-  >
-    <img src="https://img.shields.io/github/license/SecureSECODAO/diamond-governance" />
-  </a>
-  <a
-    href="https://github.com/SecureSECODAO/diamond-governance/graphs/contributors"
-    alt="Contributors"
-  >
-    <img src="https://img.shields.io/github/contributors/SecureSECODAO/diamond-governance" />
-  </a>
-  <a
-    href="https://github.com/SecureSECODAO/diamond-governance/pulse"
-    alt="Activity"
-  >
-    <img src="https://img.shields.io/github/commit-activity/m/SecureSECODAO/diamond-governance" />
-  </a>
-  <a
-    href="https://github.com/SecureSECODAO/diamond-governance/actions"
-    alt="Actions"
-  >
-    <img src="https://github.com/SecureSECODAO/diamond-governance/actions/workflows/unit-test-pipeline.yml/badge.svg?branch=main" />
-  </a>
+  <a
+    href="https://github.com/SecureSECODAO/diamond-governance/blob/master/LICENSE"
+    alt="License"
+  >
+    <img src="https://img.shields.io/github/license/SecureSECODAO/diamond-governance" />
+  </a>
+  <a
+    href="https://github.com/SecureSECODAO/diamond-governance/graphs/contributors"
+    alt="Contributors"
+  >
+    <img src="https://img.shields.io/github/contributors/SecureSECODAO/diamond-governance" />
+  </a>
+  <a
+    href="https://github.com/SecureSECODAO/diamond-governance/pulse"
+    alt="Activity"
+  >
+    <img src="https://img.shields.io/github/commit-activity/m/SecureSECODAO/diamond-governance" />
+  </a>
+  <a
+    href="https://github.com/SecureSECODAO/diamond-governance/actions"
+    alt="Actions"
+  >
+    <img src="https://github.com/SecureSECODAO/diamond-governance/actions/workflows/unit-test-pipeline.yml/badge.svg?branch=main" />
+  </a>
 </p>
 
 # Diamond Governance
-Diamond Governance is a very flexible plugin for AragonOSx. It acts as a bridge between the Aragon framework and [ERC-2535 facets](https://eips.ethereum.org/EIPS/eip-2535). Multiple facets to customize the plugin are included in the project. Currently the plugin is only launched on mumbai, but will launch on polygon before the end of June. A website to easily create a DAO with Diamond Governance and configure it to your usecase is planned, but not yet finished.
+Diamond Governance is a very flexible plugin for AragonOSx. It acts as a bridge between the Aragon framework and [ERC-2535 facets](https://eips.ethereum.org/EIPS/eip-2535). Multiple facets to customize the plugin are included in the project. Currently, the plugin is only available on Mumbai, but it will launch on Polygon before the end of June. A website to easily create a DAO with Diamond Governance and configure it to your use case is planned but not yet finished.

```

Comment:
"A website to easily create a DAO with Diamond Governance and configure it to your use case is planned but not yet finished."
It is?

Reply:
You may ignore this